### DISCUSSION TITLE
Setup varying Wild Magic Surge triggers dependent on spell level

### DISCUSSION THREAD

**Discussion by LastRed** [johnnolan/wild-magic-surge-5e#28] [Q&A] [answered]
How is this done? 
Core: 0.8.8
DnD5e: 1.4.2
Module: 0.8.6
![image](https://user-images.githubusercontent.com/76203228/130339889-41a53aae-0873-40d4-8098-0c6cacd2cb63.png)

**Comment by johnnolan** [comment, marked as answer]
Hi, I have updated the readme to explain where a little better https://github.com/johnnolan/wild-magic-surge-5e#setup-varying-wild-magic-surge-triggers-dependent-on-spell-level

Once you enable the option you can use the level options to set a specific role equation for each spell level. Shout if that helps

[![Setup varying Wild Magic Surge triggers dependent on spell level](https://raw.githubusercontent.com/johnnolan/wild-magic-surge-5e/main/images/varying-triggers.jpg)](https://raw.githubusercontent.com/johnnolan/wild-magic-surge-5e/main/images/varying-triggers.jpg)
